R-packages/covidcast/R/covidcast.R

```diff
@@ -515,32 +515,68 @@ summary.covidcast_meta = function(object, ...) {
   invisible(df)
 }
 
-##########
+# Find the maximum number of rows the API is expected to return per day of data
+max_geo_values <- function(data_source, signal, geo_type) {
+  meta_info <- covidcast_meta()
+  max_locations <- meta_info[meta_info$data_source == data_source &
+                              meta_info$signal == signal &
+                              meta_info$geo_type == geo_type, ]$num_locations
+  if (length(max_locations) == 0) {
```

Comment:
I believe it's a pre-emptive fix for a similar effect of #270, but yeah, a comment would be good here

Reply:
That's right, it's a defensive measure against signals that aren't documented in `covidcast_meta` and using the max as an upper bound when it can't be found. Added a comment.